app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreenTest.kt

```diff
@@ -1,30 +1,72 @@
 package com.kks.nimblesurveyjetpackcompose.ui.presentation.splash
 
-import androidx.compose.ui.test.assertIsDisplayed
-import androidx.compose.ui.test.junit4.createAndroidComposeRule
-import androidx.compose.ui.test.junit4.createComposeRule
-import androidx.compose.ui.test.onNodeWithContentDescription
-import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
 import com.kks.nimblesurveyjetpackcompose.R
 import com.kks.nimblesurveyjetpackcompose.base.BaseAndroidComposeTest
-import com.kks.nimblesurveyjetpackcompose.ui.presentation.main.MainActivity
+import com.kks.nimblesurveyjetpackcompose.di.RepositoryModule
+import com.kks.nimblesurveyjetpackcompose.model.ResourceState
+import com.kks.nimblesurveyjetpackcompose.model.response.LoginResponse
+import com.kks.nimblesurveyjetpackcompose.repo.login.LoginRepo
 import com.kks.nimblesurveyjetpackcompose.ui.theme.NimbleSurveyJetpackComposeTheme
-import com.ramcosta.composedestinations.navigation.EmptyDestinationsNavigator
+import com.kks.nimblesurveyjetpackcompose.viewmodel.splash.SplashViewModel
+import dagger.hilt.android.testing.BindValue
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import dagger.hilt.android.testing.UninstallModules
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
-import org.junit.runner.RunWith
 
-@RunWith(AndroidJUnit4::class)
+@UninstallModules(RepositoryModule::class)
+@HiltAndroidTest
 class SplashScreenTest : BaseAndroidComposeTest() {
 
+    @get:Rule(order = 0)
+    var hiltRule = HiltAndroidRule(this)
+
+    @BindValue
+    @JvmField
+    val splashViewModel: SplashViewModel = SplashViewModel(Dispatchers.IO)
+
+    @Before
+    fun setup() {
+        hiltRule.inject()
```

Comment:
If we put it here there's no need to repeat it on every test below 😉 
```suggestion
        hiltRule.inject()
        setupSplashComposeRule()
```